torch/csrc/jit/codegen/cuda/interface.cpp

```diff
@@ -706,7 +706,7 @@ RegisterOperators reg_reshape_copy({
             pop(stack, self, shape);
             push(
                 stack,
-                at::native::reshape(self.toTensor(), shape.toIntVector()));
+                at::reshape(self.toTensor(), shape.toIntVector()));
```

Comment:
with python reshape don't need this

Reply:
we're not doing python reshape, no? But we'll shim this, so I think we still don't need it.